docs/packaging/packaging-with-visual-studio.md

```diff
@@ -0,0 +1,81 @@
+## Overview
+This page will cover everything needed to package an application into an isolated Win32 app. 
+
+## Required software pre-requisites:
+* **Visual Studio** version 17.10.2 or higher
+
+## Step 1 - Install the required workloads in Visual Studio including Windows 11 SDK 10.0.26100.0 (or greater)
+Open the Visual Studio Installer and Modify the existing installation. 
+On the Workloads tab check the "Windows application development" box for development in C# 
+(and optionally check the "C++ WinUI app development tools" box for development in C++). 
+The Windows 11 SDK 10.0.26100.0 (or greater) is also required.
+
+![image](images/vs/01-VisualStudio-Required-Workloads.png)
+
+## Step 2 - Create your app project
+In Visual Studio, create a new C# or C++ project using the template *Blank App, 
+Packaged with Windows Application Packaging Project (WinUI 3 in Desktop)*. 
+Click "Create" and then select "10.0.26100.0" (or later) for Target Platform Version (TPV) 
+and Target Platform Minimum Version (TPMinV). 
+
+![image](images/vs/02-Blank-App-Packaged-With-Windows-Application-Packaging-Project.png)
```

Comment:
There are two issues with the images.

1. They are currently stick together without any margins. That could be easily solved by adding an extra blank line between the images.
2. The high-res images are nice, but some of them are too big which makes the docs very difficult to read. We should make the letters on the images readable, but some images are occupying a full page or even more. The general guidance should be making the letters on the images about the same size of the actual texts in the docs (it doesn't have to be super accurate). This way the user won't experience the "giant image shock".

To limit the size of the image, convert
```md
![image](images/vs/02-Blank-App-Packaged-With-Windows-Application-Packaging-Project.png)
```
to
```md
<img src="images/vs/02-Blank-App-Packaged-With-Windows-Application-Packaging-Project.png" alt="image" height="500"/>
```

Limiting `height` might be more intuitive in most cases, but `width` could also work.